src/loader.js

```diff
@@ -107,7 +107,7 @@ export function pitch(request) {
         }
         this[NS](text, query);
         if (text.locals && typeof resultSource !== 'undefined') {
-          resultSource += `\nmodule.exports = ${JSON.stringify(text.locals)};`;
+          resultSource += `\nexport default ${JSON.stringify(text.locals)};`;
```

Comment:
iirc, this is going to suffer from the same kind of issue that file-loader had.

Until the css-loader/new-loader this is only going to break things.

Reply:
@alexlur - etwp will eventually be retired & it's functionality will be rolled back into webpack where it belongs.

This is a change that has to happen across the entire style sheet chain simultaneously and is slated to happen when it's all reworked to pull some of the missing style functionality back into webpack.